projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -234,25 +309,34 @@ public function render_and_die() {
 		$this->render_recovery_form();
 	}
 
+	/**
+	 * Render the blocked login message.
+	 */
 	public function render_blocked_login_message() {
 		$this->protect_die( $this->get_html_blocked_login_message() );
 	}
 
-	function process_recovery_email() {
-		$sent = $this->send_recovery_email();
+	/**
+	 * Process sending a recovery email.
+	 */
+	public function process_recovery_email() {
+		$sent               = $this->send_recovery_email();
 		$show_recovery_form = true;
 		if ( is_wp_error( $sent ) ) {
 			if ( 'email_already_sent' === $sent->get_error_code() ) {
 				$show_recovery_form = false;
 			}
-			$this->protect_die( $sent,null,true, $show_recovery_form );
+			$this->protect_die( $sent, null, true, $show_recovery_form );
 		} else {
 			$this->render_recovery_success();
 		}
 	}
 
-	function send_recovery_email() {
-		$email = isset( $_POST['email'] ) ? $_POST['email'] : '';
+	/**
+	 * Send the recovery form.
+	 */
+	public function send_recovery_email() {
+		$email = isset( $_POST['email'] ) ? $_POST['email'] : ''; // phpcs:ignore WordPress.Security.NonceVerification.Missing -- no changes to the site, sends recovery email if email matches a user on the site.
```

Comment:
would it be worth mentioning that `send_recovery_email` is only triggered after another nonce check (`bypass-protect`)?